pages/manual/planet.md

```diff
@@ -160,7 +160,7 @@ Alternatively, click on your course on the courses list to get the course page t
 
 Alternatively, click on your course on the courses list to get the course page then click the "Leave" button near the top-right of the course page.
 
-4. To change a collection, select the checkbox next to a course then click "Change Collections". To learn what a collection is, click [here](#Course_Collections).
+4. To change a collection, select the checkbox next to a course then click "Change Collections". To learn what a collection is, click [here](https://planetlearning.github.io/#!./pages/micromaster/../manual/planet.md#Courses).
```

Comment:
This already seems to be working before. This is internal link so if you try to open on different tab then it will not wokr

Reply:
Ah - I see now!  Thanks.